components/file_system/src/io_stats/proc.rs

```diff
@@ -140,14 +129,17 @@ impl AtomicIOBytes {
 /// Flushes the local I/O stats to global I/O stats.
 #[inline]
 fn flush_thread_io(sentinel: &mut LocalIOStats) {
-    if let Some(io_bytes) = sentinel.id.fetch_io_bytes() {
+    if let Ok(io_bytes) = sentinel.id.fetch_io_bytes() {
         GLOBAL_IO_STATS[sentinel.io_type as usize]
             .fetch_add(io_bytes - sentinel.last_flushed, Ordering::Relaxed);
         sentinel.last_flushed = io_bytes;
     }
 }
 
 pub fn init() -> Result<(), String> {
+    ThreadID::current()
+        .fetch_io_bytes()
+        .map_err(|e| format!("failed to fetch I/O bytes from proc: {}", e))?;
```

Comment:
Should we log a warning here?

Reply:
It will warn in `server::init_io_utility`.